filters/openpolicyagent/openpolicyagent.go

```diff
@@ -730,7 +766,7 @@ func (opa *OpenPolicyAgentInstance) Config() *config.Config { return opa.opaConf
 
 // DistributedTracing is an implementation of the envoyauth.EvalContext interface
 func (opa *OpenPolicyAgentInstance) DistributedTracing() opatracing.Options {
-	return opatracing.NewOptions(opa)
+	return opatracing.NewOptions(opa.registry.tracer, opa.bundleName, opa.manager)
```

Comment:
I think this should wrap params via `With*`

Reply:
Good catch. Changed this...